### ISSUE TITLE
[BUG] Can't mount tmpfs volume

### ISSUE THREAD

**Issue by ChoffaH** [BastilleBSD/bastille#504]
**Describe the bug**
Trying to mount a tmpfs volume fails.
This is what my Bastillefile contains:
```
MOUNT tmpfs tmp tmpfs rw,nosuid,mode=01777 0 0
```

And that results in the following error:
```
Detected invalid host path or incorrect mount type in FSTAB.
Format: /host/path jail/path nullfs ro 0 0
Read: tmpfs tmp tmpfs rw,nosuid,mode=01777 0 0
Failed to execute command: mount
```

**Bastille and FreeBSD version (paste ``bastille -v && freebsd-version -kru`` output)**
```
0.9.20211225
13.0-RELEASE-p4
```

**How did you install bastille? (port/pkg/git)**
pkg

**Steps to reproduce?**
Just add the above config to a jail or run `bastille mount JAIL tmpfs tmp tmpfs rw,nosuid,mode=01777 0 0`

**Expected behavior**
tmpfs volume mounted inside jail

**Comment by bmac2** [comment]
@ChoffaH and @zilti  is this one still an issue??? if so what is the steps so I can reproduce????

**Comment by ChoffaH** [comment]
> @ChoffaH and @zilti is this one still an issue??? if so what is the steps so I can reproduce????

Yes, I can still reproduce this by creating a new jail and running the command under "Steps to reproduce" against it.

**Comment by bmac2** [comment]
@ChoffaH thank you .   Will keep in queue as a bug to be fixed.

**Comment by bmac2** [comment]
https://forums.freebsd.org/threads/mount-tmpfs-inside-jail.47691/

posting on how to mount a tmpfs inside a jail.  Will look at writing up how to do this and add to the docs

**Comment by tschettervictor** [comment]
@ChoffaH can you give #778 a try?

**Comment by tschettervictor** [comment]
Can you test #800?
@ChoffaH

**Comment by tschettervictor** [comment]
Fix merged.
